Author: Gerfried Fuchs <rhonda@debian.at>	vim:ft=diff:
Description: Do not remove a file from the system on install

Index: beep-1.2.2/Makefile
===================================================================
--- beep-1.2.2.orig/Makefile
+++ beep-1.2.2/Makefile
@@ -15,5 +15,5 @@ beep : beep.c
 
 install : 
 	cp ${EXEC_NAME} ${INSTALL_DIR}
-	rm -f /usr/man/man1/beep.1.bz2
+	# rm -f /usr/man/man1/beep.1.bz2
 	cp ${MAN_FILE} ${MAN_DIR}
